desktop/apps/auction/client/actions.js

```diff
@@ -29,31 +29,31 @@ export const UPDATE_SORT = 'UPDATE_SORT'
 export const UPDATE_TOTAL = 'UPDATE_TOTAL'
 
 // Action creators
-export function decrementFollowedArtistsPage() {
+export function decrementFollowedArtistsPage () {
```

Comment:
Thank you for fixing this!! (although side note, having the space looks so odd to me)